docs/reference/analysis/tokenfilters/stemmer-tokenfilter.asciidoc

```diff
@@ -4,189 +4,278 @@
 <titleabbrev>Stemmer</titleabbrev>
 ++++
 
-// Adds attribute for the 'minimal_portuguese' stemmer values link.
-// This link contains ~, which is converted to subscript.
-// This attribute prevents that substitution.
-// See https://github.com/asciidoctor/asciidoctor/wiki/How-to-prevent-URLs-containing-formatting-characters-from-getting-mangled
-:min-pt-stemmer-values-url: http://www.inf.ufrgs.br/~buriol/papers/Orengo_CLEF07.pdf
+Provides algorithmic stemming for several languages, some with additional
```

Comment:
I can xref algorithmic stemming with https://github.com/elastic/elasticsearch/pull/55156/files#diff-8dbd6670213faa5eccc209176d0dd22bR33 when #55156 is merged.